### DISCUSSION TITLE
Gruntwork help with drift detection

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#482] [Help]
Does Gruntwork have any tooling or guidance around tooling to manage drift detection outside of what Terraform can already do? 

---

<ins datetime="2022-06-22T19:57:14Z">
  <p><a href="https://support.gruntwork.io/hc/requests/108817">Tracked in ticket #108817</a></p>
</ins>

**Comment by yorinasub17** [comment]
Unfortunately, we don't have any tooling beyond what Terraform offers to help with drift detection.
